core/src/find_packet_sender_stake_stage.rs

```diff
@@ -27,27 +29,65 @@ thread_local!(static PAR_THREAD_POOL: RefCell<ThreadPool> = RefCell::new(rayon::
                     .build()
                     .unwrap()));
 
+pub type FindPacketSenderStakeReceiver = Receiver<PacketBatch>;
```

Comment:
heh this might be a bit confusing, but convention in the code base right now would make `FindPacketSenderStakeReceiver = Receiver<Vec<PacketBatch>>` which would then be used in sigverify here: https://github.com/solana-labs/solana/blob/46dfbc91ce6f004a46e8d9695bd0bca23eea846e/core/src/sigverify_stage.rs#L195

Reply:
meanwhile we can just import `PacketBatchReceiver` from `streamer.rs` via `use solana_streamer::streamer::PacketBatchReceiver`  instead of `Receiver<PacketBatch>` here

Reply:
Yea, I messed it up. 😮‍💨  Corrected it now, using `streamer::PacketBatchReceiver` here, and `find_packet_sender_stake_stage::FindPacketSenderStakeReceiver` in sigverify_stage.